commit eae43a50c6252d487b65124ad20b0019c320accc (refs/changes/14/2914/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-06T10:26:25-08:00 (1 year, 11 months ago)
    
    DAPI-346 sdc-designation tests are failing in nightly-1: tests 'allocate 1' and 'allocate 2'

diff --git a/test/integration.test.js b/test/integration.test.js
index ff084c2..edb597f 100644
--- a/test/integration.test.js
+++ b/test/integration.test.js
@@ -190,7 +190,7 @@ test('allocate 1', function (t) {
 			}
 		}, {
 			/* JSSTYLED */
-			step: 'Servers which meet image manifest platform requirements',
+			step: 'Servers which meet image manifest and package platform requirements',
 			remaining: [
 				'00000000-0000-0000-0000-00259094373c',
 				'00000000-0000-0000-0000-0025909437d4'
@@ -377,7 +377,7 @@ test('allocate 2', function (t) {
 			}
 		}, {
 			/* JSSTYLED */
-			step: 'Servers which meet image manifest platform requirements',
+			step: 'Servers which meet image manifest and package platform requirements',
 			remaining: [
 				'00000000-0000-0000-0000-00259094373c',
 				'00000000-0000-0000-0000-0025909437d4'
